src/main/java/com/amazon/opendistroforelasticsearch/security/support/Base64Helper.java

```diff
@@ -77,6 +78,7 @@
         InetSocketAddress.class,
         Pattern.class,
         User.class,
+        UserInjector.InjectedUser.class,
```

Comment:
Consider moving `User.class` from `SAFE_CLASSES` to `SAFE_ASSIGNABLE_FROM_CLASSES`.